src/appleseed/renderer/kernel/shading/closures.h

```diff
@@ -40,6 +40,7 @@
 #include "renderer/modeling/bsdf/specularbrdf.h"
 #include "renderer/modeling/bsdf/specularbtdf.h"
 #include "renderer/modeling/bsdf/velvetbrdf.h"
+#include "renderer/modeling/bssrdf/dipolebssrdf.h"
```

Comment:
Not sure we want to include this file, `DipoleBSSRDF` is a "technical" base class, not an actual model.